projects/core/src/global-message/http-interceptors/handlers/http-error.handler.ts

```diff
@@ -44,6 +44,15 @@ export abstract class HttpErrorHandler implements Applicable {
     return errorResponse.status === this.responseStatus;
   }
 
+  /**
+   * Converts error description to translation key format.
+   *
+   * Example: 'User is disabled' will be transformed to 'user_is_disabled'.
+   */
+  createErrorTranslationKey(reason: string): string {
```

Comment:
`getErrorTranslationKey` could be a better name for this method as it does not really create anything.

Reply:
Right, I changed the method name.